shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1488,6 +1497,24 @@ private void createAndSendWindowChangeEvent(@NonNull SemanticsNode route) {
         sendAccessibilityEvent(event);
     }
 
+    /**
+     * Factory method that creates a {@link AccessibilityEvent#TYPE_WINDOW_CONTENT_CHANGED} and sends
+     * the event to Android's accessibility system.
+     *
+     * It sets the content change types to {@link AccessibilityEvent#CONTENT_CHANGE_TYPE_SUBTREE}
+     *
+     * The given {@code virtualViewId} should be a {@link SemanticsNode} below
+     * which the content changed.
+     */
+    private void createAndSendWindowContentChangeEvent(int virtualViewId) {
+        AccessibilityEvent event = obtainAccessibilityEvent(
+            virtualViewId,
+            AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED
+        );
+        event.setContentChangeTypes(AccessibilityEvent.CONTENT_CHANGE_TYPE_SUBTREE);
```

Comment:
Technically, we could probably detect if something more specific then "the entire subtree" has changed...

For what do we need this? Does this change the behavior?

Reply:
A guy on the TalkBack team mentioned that we should be setting this, so I added it.  It doesn't appear to change behavior.